src/test/mods/paramHelper.spec.ts

```diff
@@ -277,7 +286,7 @@ describe("paramHelper", () => {
       expect(promptStub.callCount).to.equal(2);
       expect(promptStub.firstCall.args).to.eql([
         {
-          default: "default",
+          default: "test-proj",
```

Comment:
Is there any test where this value (`NEW_PARAMETER`) is populated in the result from `paramHelpers.promptForNewParams` using `PROJECT_ID`?

Reply:
Missed this, sorry for the delay. I'm not sure of the best way to do this - we rely on inquirer to return the default if the user hits enter, but we are stubbing out prompt/inquirer in these tests. I added a test that just sets the stub to return the default, but it seems a bit trivial - do you think i need to change what I am stubbing to properly test this?

Reply:
Can `promptStub.withArgs` help in this? Can the stub change based on input?

Come chat with me if you need and we can walk through it.